app/src/main/java/org/piwigo/ui/settings/SettingsActivity.java

```diff
@@ -70,4 +72,45 @@ public void onNothingSelected(AdapterView<?> parent) {
             }
         });
     }
+
+    private void initializeBrightnessSeekBar(){
```

Comment:
ah, brightness adjustment needs the WRITE_SETTINGS permission. But I think the app should not mess with this system setting. Why not leave this just up to the system?

Reply:
yup, will remove